### PR TITLE
Charles development

### PR DESCRIPTION
**Pull request by chuckluv**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,14 +5,42 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+table, th, td {
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/index.html]: You can probably put this in your style.css

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,14 +5,42 @@
...
+</style>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Charles Lovett's Portfolio</h1>
+      <p>This Portfolio contians my life growing up and during college. Hope you enjoy learning a little bit about me.</p>
```
Comment (bsidhom) [inline portfolio/src/main/webapp/index.html]: "contians" -> "contains"

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ['Favorite TV show is Game of Thrones', 'Favorite Sport is Football', 'I have fractured every finger on my left hand', 'I have played the double bass since Third Grade'];
```
Comment (bsidhom) [inline portfolio/src/main/webapp/script.js]: Ouch! Sorry to hear about your left hand!

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,14 @@
...
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const funFact = funFacts[Math.floor(Math.random() * funFacts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('greeting-container');
```
Comment (bsidhom) [inline portfolio/src/main/webapp/script.js]: You may want to change the name of this containing element (maybe 'fun-fact-container' or similar).

**Review by bsidhom — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -5,14 +5,42 @@
...
+      <p>Real Quick if you want to know some fun facts about me press the button below:</p>
+      <button onclick="addRandomFunFact()">Fun Facts</button>
       <div id="greeting-container"></div>
+
+    
+    </div>
+    <div id='choice' >
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/index.html]: Please use double quotes for attribute values per the [Google style guide](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks).

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (KoopaKing) [inline portfolio/src/main/webapp/script.js]: You might reformat this so the line isn't long. Google javascript style guide (and most Google style guides) recommend [80 char max limit](https://google.github.io/styleguide/jsguide.html#formatting-column-limit) per line unless there is a functional reason not to break it.

**Review by KoopaKing — CHANGES_REQUESTED** [review]
A few more changes, you can keep them in this pull request I think by adding additional commits into it, or updating the commits included in the change (maybe squashing the fixes into the existing commits).

Also high level comments:
 - you might want to add some content to your new HTML files, even just a TODO or WIP comment
 - you might want to remove one of the duplicate pictures (Collegelife) since it seems like you will probably only use one

Let me know if you want help updating the pull request or squashing the commits.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -2,8 +2,20 @@
...
   margin-right: auto;
   width: 650px;
+  text-align: center;
 }
 
-#greeting-container {
+#fun-fact-container {
   margin-top: 20px;
 }
+
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/style.css:13]: super nit: there is a lot of unnecessary whitespace.

File: portfolio/src/main/webapp/college.html [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<!--#Do a timeline of important events
+#start with after highschool graduction
+#conclude with start of STEP intern ship-->
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/college.html]: This newline thing is super annoying. No need to fix now though. This isn't blocking.

**Review by KoopaKing — APPROVED** [review]

File: portfolio/src/main/webapp/college.html [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+              However, I think I have already done way more than I expected to do by now. A quick overview I currently attend the illustrious
+              Howard University as a rising junior computer science major.
+          </p>
+      </div>
+
+
+           <hr>
+      <div class="main">
+
+<h2>Profession Times</h2>
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/college.html]: Not sure why the indentation changes suddenly here.

File: portfolio/src/main/webapp/college.html [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+     <video controls>
+         <source src="images/proto.MP4" type="video/mp4">
+         
+     </video>
+     
+      <p>The prototype for the app Secure that my group made</p>
+    </div>
+   </div>
+
+
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/college.html]: Can you clean up the whitespace in this file a bit?

File: portfolio/src/main/webapp/growing-up.html [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+      <p>I was made a tackle for loss in the state championship game. Unfortunatly 
+                we still lost :(</p>
+    </div>
+    </div>
+     <div class="column">
+    <div class="content">
+     <img src="images/football1.PNG" alt="Mountains" style="width:100%">
+     
+      <p>I was about to dive for the game winning touchdown. I still have the ball 
+                in hold in the picture at my house.</p>
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/growing-up.html]: should this start with "I'm holding"?

File: portfolio/src/main/webapp/style.css [hunk 11]
```diff
@@ -2,8 +2,104 @@
...
+/* Responsive layout - makes the two columns stack on top of each other instead of next to each other */
+@media screen and (max-width: 600px) {
+  .column {
+    width: 100%;
+  }
+}
+
+
+
+
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/style.css]: whitespace issue again.

File: portfolio/src/main/webapp/growing-up.html [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+    </div>
+    </div>
+
+
+
+
+
+
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/growing-up.html]: whitespace issue again.

File: portfolio/src/main/webapp/growing-up.html [hunk 13]
```diff
@@ -0,0 +1,231 @@
...
+   </div>
+  
+</div>
+
+</div>
+
+      
+     
+  </body>
+</html>
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/growing-up.html:223]: no EOF.

File: portfolio/src/main/webapp/college.html [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+
+
+
+
+
+
+
+</html>
```
Comment (KoopaKing) [inline portfolio/src/main/webapp/college.html:154]: no EOF

**Review by KoopaKing — COMMENTED** [review]
So this is great, I love the amount of content you got done.

Can you clean up the whitespace and formatting in a single commit though before merging?

**Review by bsidhom — APPROVED** [review]
LGTM

**Pull request merged**